src/taco-livereload/lib/utils/homePage/bower_components/q/.npmignore

```diff
@@ -0,0 +1,23 @@
+node_modules
```

Comment:
Is this correct? We want to not package up the dependencies of the q package used by bower? Do we `bower install` at runtime?